interpreter/cling/include/cling/Interpreter/RuntimeUniverse.h

```diff
@@ -175,6 +175,11 @@ extern "C" {
   /// definition of the exceptions from the RuntimeUniverse and allows us to
   /// run cling in -no-rtti mode.
   ///
+
+  void cling__runtime__internal__check__and__throw__if__invalid(void* Sema,
```

Comment:
The `__` was meant to signal `::` in C :-) Maybe rename it to `cling_runtime_internal_ThrowOnInvalidPointer`?